public/styles/_releases.scss

```diff
@@ -13,75 +17,87 @@
   &:hover {
     text-decoration: none;
     color: $main-color-strong;
-    background-color: $main-bg-color-shade;
+    background-color: $main-bg-color-shade !important;
   }
 }
 
+.r-class {
+  // Magic numbers.
+  margin-right: 128px;
+  margin-left: 128px;
+}
+
 .release-version-filter .active {
   text-decoration: underline;
 }
 
 .r-menu {
-  display: flex;
-  flex-direction: column;
-  padding-left: 8px;
-  list-style: none;
+  display: flex !important;
+  flex-direction: column !important;
+  padding-left: 8px !important;
+  list-style: none !important;
 
   .menu-item {
-    font-size: 21px;
+    font-size: 21px !important;
     // Remove border radius for active element.
-    border-radius: 0;
-    position: relative;
-    display: block;
-    padding: 12px 15px;
-    border-bottom: 1px solid $border-color;
-
-    &:first-child {
-      border-top: 0;
-      border-top-right-radius: $border-radius;
-      border-top-left-radius: $border-radius;
-    }
+    border-radius: 0 !important;
 
-    &:last-child {
-      border-bottom: 0;
-      border-bottom-right-radius: $border-radius;
-      border-bottom-left-radius: $border-radius;
+    .octicon {
+      font-size: 20px !important;
+      padding-right: 8px !important;
     }
 
     &:hover {
-      text-decoration: none;
-      background-color: $gray-0;
-    }
-
-    &.selected {
-      font-weight: bold;
-      color: $gray-7;
-      cursor: default;
-      background-color: $white;
-    }
-
-    .octicon {
-      font-size: 20px;
-      padding-right: 8px;
+      cursor: pointer !important;
+      background-color: #fcfcfc !important;
     }
 
     p {
-      font-size: 15px;
+      font-size: 15px !important;
       // margin-left: 23px;
     }
   }
-}
 
-.r-flex {
-  display: flex;
-  padding-right: 16px;
-  padding-left: 16px;
+  ul {
+    margin-bottom: 1em !important;
+    margin-left: 1.5em !important;
+    padding-left: 0;
+    padding-top: 0;
+  }
 }
 
 .releases {
-  width: 1012px;
+  width: 1280px;
   flex: 1;
 
+  h1 {
+    font-size: $h0-size-mobile !important;
+    font-weight: $font-weight-light !important;
+
+    @include breakpoint(md) { font-size: $h0-size !important; }
+  }
+
+  h2 {
+    font-size: $h1-size-mobile !important;
+    font-weight: $font-weight-light !important;
+
+    @include breakpoint(md) { font-size: $h1-size !important; }
+  }
+
+  h3 {
+    font-size: $h2-size-mobile !important;
+    font-weight: $font-weight-light !important;
+
+    @include breakpoint(md) { font-size: $h2-size !important; }
+  }
+
+  h4 {
+    font-size: $h3-size-mobile !important;
+    font-weight: $font-weight-light !important;
+
+    @include breakpoint(md) { font-size: $h3-size !important; }
+  }
+
```

Comment:
Can get rid of all the `!important` if we wrap this entire block in `.markdown-body`.